config/default.json

```diff
@@ -79,6 +80,13 @@
     "hidePoweredBy": {
       "setTo": "XUI Server 1.0"
     }
+  },
+  "redis": {
+    "host": "xui-ao-webapp-prod.redis.cache.windows.net",
```

Comment:
I have removed these as it should be coming from vault directly

Reply:
Also we use the connection string now and not an object key/value pair

Reply:
Done